src/templates/static/get_started/binary_options_mt5.jsx

```diff
@@ -38,7 +38,6 @@ const BinaryOptionsForMT5 = () => (
                         <li>{it.L('Trade binary options and CFDs from the same powerful interface')}</li>
                         <li>{it.L('Use comprehensive technical analysis tools to predict price movements')}</li>
                         <li>{it.L('Monitor your trades easily with real-time visualisations')}</li>
-                        <li>{it.L('Access your trading history through our Expert Advisors (EA)')}</li>
```

Comment:
this whole file should be removed since we won't be offering binary options on mt5 anymore, right @aminroosta ?

Reply:
@negar-binary If this page is only  describing binary_options on MT5 then yes.

However is see some text about CFDs on mt5, which we should keep.